.github/workflows/lint.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches: ["main"]
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - review_requested
```

Comment:
The other types make sense to me, but does `review_requested` mean that we'll re-run workflows every time a review is requested?

Also, if this is needed in order for the workflows in the Engine Bump PR to run, then we also need this in at least publish.yml as we have some tests that run there.

Reply:
> The other types make sense to me, but does `review_requested` mean that we'll re-run workflows every time a review is requested? I saw your comment about this, but will this affect every other PR too?

:face_exhaling: I accidentally copied the wrong event type. The one that should be here is `ready_for_review` so checks are also triggered when moving PR's from draft to ready. That's the original intention since the flow now opens the bump PR in `draft`

Reply:
FWIW `opened` `synchronized` and `reopened` are the default types, the additional one here is `ready_for_review` which is the one that will make the workflow triggers after the bump PR is in such state.